notebooks/WFC3/calwf3_recalibration/calwf3_recal_tvb.ipynb

```diff
@@ -55,20 +55,18 @@
     "This notebook assumes you have created the virtual environment in [WFC3 Library's](https://github.com/spacetelescope/WFC3Library) installation instructions.\n",
```

Comment:
<p>Thanks for your work. I've reviewed the current approach for moving files, and I have a suggestion that might streamline the process, especially considering the potential for repeated runs of the notebook. To ensure that we efficiently handle files that may already exist in the destination directory, I propose the following code: </p><p> </p><pre class="ql-syntax" spellcheck="false">os.makedirs('orig/', exist_ok=True)

for file_pattern in ['ibohbf*_ima.fits', 'ibohbf*_flt.fits', 'ibohbf*_drz.fits']:
&nbsp;&nbsp;for file in glob.glob(file_pattern):
&nbsp;&nbsp;&nbsp;&nbsp;destination_path = os.path.join('orig', os.path.basename(file))
&nbsp;&nbsp;&nbsp;&nbsp;if os.path.isfile(destination_path):
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;os.remove(destination_path)
&nbsp;&nbsp;&nbsp;&nbsp;shutil.move(file, destination_path)&nbsp;&nbsp;
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>